tests/VisualStudio.AndroidX.Migration/Test/VisualStudio.Tests.csproj

```diff
@@ -1,91 +1,29 @@
-    <ReferencePath>$(ProgramFiles)\Common Files\microsoft shared\VSTT\$(VisualStudioVersion)\UITestExtensionPackages</ReferencePath>
-    <IsCodedUITest>False</IsCodedUITest>
-    <TestProjectType>UnitTest</TestProjectType>
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
+    <AssemblyName>VisualStudio.AndroidX.Migration.Tests</AssemblyName>
+
+    <IsPackable>false</IsPackable>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="NamespaceTests.cs" />
-    <Compile Include="NameTests.cs" />
-    <Compile Include="NullProgress.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="ResolverTests.cs" />
-    <Compile Include="TestBase.cs" />
-    <Compile Include="ProjectTests.cs" />
-  </ItemGroup>
+
   <ItemGroup>
-    <Content Include="Assemblies\AndroidX\Xamarin.AndroidX.AppCompat.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Assemblies\AndroidX\Xamarin.AndroidX.Leanback.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Assemblies\AndroidX\Xamarin.AndroidX.Lifecycle.Common.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Assemblies\Android\Xamarin.Android.Arch.Lifecycle.Common.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Assemblies\Android\Xamarin.Android.Support.v17.Leanback.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Assemblies\Android\Xamarin.Android.Support.v7.AppCompat.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
+    <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="runtime" Version="16.0.461" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.0.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Workspaces.MSBuild" Version="3.0.0" />
+    <PackageReference Include="SQLitePCLRaw.bundle_green" Version="1.1.14" />
```

Comment:
For some reason, the `Microsoft.CodeAnalysis.CSharp.Workspaces` package was not pulled in automatically from Core... And then I find that it has a dependency on SQLite, but doesn't pull that in either... So add it I guess?